TransformationSystem/Client/TaskManager.py

```diff
@@ -120,9 +122,13 @@ def __init__(self, transClient=None, logger=None, requestClient=None,
       logger = gLogger.getSubLogger('RequestTasks')
 
     super(RequestTasks, self).__init__(transClient, logger)
+    useCertificates = True if (bool(ownerDN) and bool(ownerGroup)) else None
 
     if not requestClient:
-      self.requestClient = ReqClient()
+      self.requestClient = ReqClient(useCertificates=useCertificates,
+                                     delegatedDN=ownerDN,
```

Comment:
Pfiou :-) Have you seen this actually used somewhere, or do you just trust the comments I've put a couple of years ago ? :D

Reply:
In fact I am not sure how this works. 
Do you set the ownerDN and the ownerGroup of the Request somewhere ? If not, it tries to get it from proxyInfo, which would not do anything now if I understand properly. 
However, with the rpcStub fix that was merged lately, I am not sure of what will happen with these delegatedDN

Reply:
Do you mean these comments? :rofl: 
https://github.com/DIRACGrid/DIRAC/blob/6926fc6a086a8bc9122f76ceda63efc3a62ab5c2/Core/DISET/private/BaseClient.py#L52-L53
I saw those, laughed, looked at git ~~blame~~ praise, tried it out and saw that it worked. Otherwise I would have given up on this route.

I also thought about the stub changes and rebased my developments yesterday, things are still working.

The ownerDN/group of the requests are (already) set in the RequestTasks TaskManager

https://github.com/DIRACGrid/DIRAC/blob/6926fc6a086a8bc9122f76ceda63efc3a62ab5c2/TransformationSystem/Client/TaskManager.py#L267-L282

Reply:
:D yes, these. 
Good that it works. Not sure what you are praising though ! :D